docs/troubleshooting.md

```diff
@@ -0,0 +1,23 @@
+
+### I cannot join a worker node
+
+In [standalone](usage-overview.md#standalone) mode, check whether the join
+token has expired. If this is the case, attempt to create a new one using
+[JoinTokenRequest](https://docs.k0smotron.io/stable/join-nodes/#join-tokens) In
```

Comment:
Can we make the sentences “In standalone...” and “In Cluster API...” in separate paragraphs?

Reply:
Yes.